site/_docs/howto.md

```diff
@@ -474,6 +474,32 @@ $ ./src/main/scripts/generate-protobuf.sh
 The following sections are of interest to Calcite committers and in
 particular release managers.
 
+## Managing Calcite repositories through GitHub (for Calcite committers)
+
+Committers have write access to Calcite's 
+[ASF git repositories](https://gitbox.apache.org/repos/asf#calcite) hosting
+the source code of the project as well as the website.
+
+All repositories present on gitbox are available on GitHub with write-access 
+enabled, including rights to open/close/merge pull requests and address issues.
+
+In order to exploit the GitHub services, committers should link their accounts
+(ASF, GitHub) via the [account linking page](https://gitbox.apache.org/setup/).
+
+Here are the steps:
+
+ * Set your GitHub ID into your [Apache ID profile](https://id.apache.org/).
```

Comment:
Set your GitHub ID into your [Apache ID profile](https://id.apache.org/)
vv
Set your GitHub username in your [Apache profile](https://id.apache.org/)